css-grid-classes/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Grid Classes</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column">
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
```

Comment:
This `column` class doesn't seem to be doing anything. Make sure you use a column class with styling that matches the situation and a name that describes its purpose.